### PR TITLE
Wishlist

### PR DESCRIPTION
**Pull request by raghav6202**
Please Review the Wishlist page 

image

### CODE REVIEW

**Review by krate27 — COMMENTED** [review]
Everything looks good, just you can use a good css code formatter.

File: Components/Cart/cart.html [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+    </div>
+
+</section>
+    </div>
+
+
+        </section>
+
+</body>
+</html>
```
Comment (pritam-kr) [inline Components/Cart/cart.html:86]: Remove these red marks by breaking line.

File: component-lib/styles.css [hunk 2]
```diff
@@ -0,0 +1,322 @@
...
+{
+    --primary-color: #63DCCD;
+    --secondary-color: #266350;
+}
+
+.logo-component
+{
+
+    height: 50px;
+    width: 60px;
```
Comment (pritam-kr) [inline component-lib/styles.css:26]: you can use rem unit for width and height too.

File: component-lib/styles.css [hunk 3]
```diff
@@ -0,0 +1,322 @@
...
+}
+
+.navigation-menu
+{
+    padding: 1.5rem 1rem;
+    /* width: 100%; */
```
Comment (pritam-kr) [inline component-lib/styles.css:32]: remove commented code

**Review by pritam-kr — APPROVED** [review]
All looks good to merge.

**Review by krate27 — COMMENTED** [review]
Approved, good to go

**Pull request merged**